low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -280,6 +289,35 @@ def get_disk_groups_info(self):
                 })
         return dg_data
 
+    def get_drives_info(self):
+        """Update and return drives information in specific format."""
+        drive_data = []
+        drives = self.get_realstor_encl_data("drives")
+        for drive in drives:
+            slot = drive.get("slot", -1)
+            if slot == -1:
```

Comment:
Any cases seen with slot = -1 ?

Reply:
I'm not sure when this case will occur. I just referred to generate_health_view script and put this.
I don't see slot=-1 in disk api response.